Cargo.toml

```diff
@@ -52,7 +52,6 @@ url = "2"
 [features]
 default = ["reqwest/default-tls"]
 rustls-tls = ["reqwest/rustls-tls", "tokio-tungstenite/rustls"]
-gaggle = []
```

Comment:
It seems we're making reference to this feature in the code still, so this is actually required. Sorry that I missed that.

Reply:
Ok, dropped it.